src/main/resources/org/datadog/jenkins/plugins/datadog/DataDogJobProperty/config.jelly

```diff
@@ -0,0 +1,14 @@
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:optionalBlock name="on" checked="${instance.on}" title="DataDog tagging" inline="true">
```

Comment:
I'd prefer that we put this in a section of its' own with the heading `Datadog Tagging`

Similar to how Slack does it
<img width="256" alt="screen shot 2016-03-11 at 2 05 44 am" src="https://cloud.githubusercontent.com/assets/1171118/13695557/f3a005d8-e72d-11e5-89c3-5597a2f13cd4.png">